packages/account/src/Components/poi/status/unsupported/unsupported.tsx

```diff
@@ -61,7 +63,9 @@ const Unsupported = ({
     if (manual) {
         if (manual.status === identity_status_codes.pending)
             return <UploadComplete is_manual_upload needs_poa={needs_poa} redirect_button={redirect_button} />;
-        else if ([identity_status_codes.rejected, identity_status_codes.suspected].includes(manual.status)) {
+        else if (
+            [identity_status_codes.rejected, identity_status_codes.suspected].some(status => status === manual.status)
+        ) {
```

Comment:
this change will have the exact same result as `includes` we can revert this or let me know if I'm missing something why this change is required?

Reply:
yes, the result is the same, but no TS errors :)
better to not revert, but to use such approach in other places as well :)

Reply:
There's no point of having typescript then as status is a constant and comparing it to a different type

Reply:
Not sure I understand you.
some() works fine in such scenarios

Reply:
![Uploading Screenshot 2023-11-17 at 13.13.59.png…]()

Reply:
<img width="1342" alt="Screenshot 2023-11-17 at 13 13 59" src="https://github.com/binary-com/deriv-app/assets/103182683/3c87b831-5303-4005-9251-a6e79be364a5">